src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -696,17 +777,26 @@ static Locale toLocale(String localeString) {
                 }
             }
         }
-        if (!isValidCountryCode) {
-            country = Locale.getDefault().getCountry();
-        }
+        return isValidCountryCode;
+    }
 
-        // language and country
-        if (parts.length == 2) {
-            return new Locale(lang, country);
+    private static boolean isAlpha(char c) {
```

Comment:
isn't it possible to use ``StringUtils.isAlpha()`` here? Or would that require to add a new dependency on OSGI level?

Reply:
Yes, it would require to add a new dependency on OSGI level.